identity-service/sequelize/migrations/20220222034002-add-cognito-flow-identity-table.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
```

Comment:
No complaint, just curious what was the design rationale behind doing this as a sep table vs. including it in the current table?

Reply:
don't want to tie identities to users

Reply:
worth a comment!